cluster-controller/src/main/java/io/strimzi/controller/cluster/operator/assembly/AbstractAssemblyOperator.java

```diff
@@ -43,7 +43,6 @@
     protected final boolean isOpenShift;
     protected final AssemblyType assemblyType;
     protected final ConfigMapOperator configMapOperations;
-
```

Comment:
Please try to avoid unnecessary whitespace changes (I'm probably guilty of this at times too)